zerver/management/commands/create_user.py

```diff
@@ -78,7 +83,8 @@ def handle(self, *args, **options):
                 full_name = input("Full name: ")
 
         try:
-            notify_new_user(do_create_user(email, initial_password(email),
+            pw = options.get('password', initial_password(email))
+            notify_new_user(do_create_user(email, pw,
```

Comment:
I merged this commit as 7a9de7766b4d0bd0ec3fac76789c08e1e47d5595.  Note the changes to the help text, and the fact that from a security perspective, this argument isn't recommended unless you have a controlled environment where nobody can get a process listing.